packages/three-vrm-node-constraint/vitest.config.mjs

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from 'vitest/config';
```

Comment:
Maybe we can use vitest workspace(projects) support to avoid duplicate vitest config?

https://vitest.dev/guide/projects